ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityRewardedAd.java

```diff
@@ -62,6 +62,11 @@
    */
   private String mPlacementId;
```

Comment:
how is the old placementId being used?  Can this be removed?

Reply:
Its the placement ID fetched from the server configuration so its not used at the same place and for the same purposes.